packages/orm/entity/Portfolio.ts

```diff
@@ -20,13 +20,13 @@ export enum DodComponent {
   OSD_PSAS = "OSD_PSAS",
   NSA = "NSA",
 }
+
 export interface IPortfolio {
   name: string;
-  description?: string;
+  administrators?: Array<string>;
 }
 
-export interface PortfolioUpdate {
-  name: string;
+export interface IPortfolioCreate extends IPortfolio {
   description: string;
```

Comment:
I saw you had `description` as optional in `IPortfolio` previously, should that have carried over to this `description`?

Reply:
Applied in a8f122e87e60c2d0aa1e8b3e00e1a38c57e7e2c5